libs/components/lists/src/lib/modules/infinite-scroll/infinite-scroll-dom-adapter.service.ts

```diff
@@ -29,8 +30,8 @@ export class SkyInfiniteScrollDomAdapterService implements OnDestroy {
    * @param elementRef The infinite scroll element reference.
    */
   public parentChanges(elementRef: ElementRef): Observable<void> {
-    this.createObserver(elementRef.nativeElement);
-    return this._parentChanges;
+    this.#createObserver(elementRef.nativeElement);
```

Comment:
Just checking, do we need to regenerate the observer every time the consumer gets the `parentChanges`?

Reply:
I don't think that's an easy change. The whole `parentChanges` observable gets unsubscribed to if `enabled` is ever set to false, but there's nothing to tell the service to dispose of the observer. So it's probably something we should look at, but maybe hold of on it for now